### PR TITLE
Update typography on headline links in MostRead

### LINKED ISSUE #3015: Change typography for headline links in MostRead & Validate UX for Burmese
**Is your feature request related to a problem? Please describe.**
After a small UX meeting, one of the last remaining issues was the typography on the headlines due to the recent change of using `SectionLabel` for the title rather than having a bespoke `MostReadTitle` component. 
The section label is significantly smaller than what we had before, so UX has requested to change the typography of the headline links to suit this better.

**Describe the solution you'd like**
Change typography of headline links from `DoublePica` to `Pica` for anything screen size below 400px and `Great Primer` for anything above 400px. A full list of changes in px can be seen below:

**Most read, title**
Double Pica
Mobile up to 599px, 20/24
Desktop from 600px, 24/28

**Headline text**
Pica
Mobile up to 399px, 16/20

**Great Primer**
Desktop from 400px, 20/24

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by hotinglok**
Resolves #3015

**Overall change:**
Changed typography of headline links in MostRead from `DoublePica` to `Pica` for screen sizes below 400px and anything above 400 (`group_2_min_width`)

**Code changes:**
- Changed typography in `StyledLink` in `MostReadItem`

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated jest tests added (for new features) or updated (for existing features)
- [x] This PR requires manual testing

**Additional context** 
Will be just a case of taking a look with UX to see that everything is fine at each breakpoint most likely.

### CODE REVIEW

File: packages/components/psammead-most-read/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 0.5.2-alpha.6 | [PR#2996](https://github.com/bbc/psammead/pull/3016) Update typography on headline links in MostRead |
```
Comment (j-pendlebury) [inline packages/components/psammead-most-read/CHANGELOG.md]:
```suggestion
| 0.5.2-alpha.7 | [PR#3016](https://github.com/bbc/psammead/pull/3016) Update typography on headline links in MostRead |
```

**Review by j-pendlebury — APPROVED** [review]
Just needs updating CHANGELOG, otherwise looks good 👍

**Review by OlgaLyubin — APPROVED** [review]
Looks good 👍

**Review by sareh — APPROVED** [review]
👍 Looks good. I see Pica for below 400px and Great Primer for above 400px.

**Pull request merged**